sdk/storage/storage-file/test/node/directoryclient.spec.ts

```diff
@@ -1,29 +1,34 @@
 import * as assert from "assert";
-import { getBSU, getUniqueName } from "../utils";
+import { getBSU } from "../utils";
 import * as dotenv from "dotenv";
-import { DirectoryClient, newPipeline, SharedKeyCredential } from "../../src";
+import { DirectoryClient, newPipeline, SharedKeyCredential, ShareClient } from "../../src";
+import { record } from "../utils/recorder";
 dotenv.config({ path: "../.env" });
 
 describe("DirectoryClient Node.js only", () => {
   const serviceClient = getBSU();
-  let shareName = getUniqueName("share");
-  let shareClient = serviceClient.getShareClient(shareName);
-  let dirName = getUniqueName("dir");
-  let dirClient = shareClient.getDirectoryClient(dirName);
+  let shareName: string;
+  let shareClient: ShareClient;
+  let dirName: string;
+  let dirClient: DirectoryClient;
 
-  beforeEach(async () => {
-    shareName = getUniqueName("share");
+  let recorder: any;
+
+  beforeEach(async function() {
+    recorder = record(this);
+    shareName = recorder.getUniqueName("share");
     shareClient = serviceClient.getShareClient(shareName);
     await shareClient.create();
 
-    dirName = getUniqueName("dir");
+    dirName = recorder.getUniqueName("dir");
     dirClient = shareClient.getDirectoryClient(dirName);
     await dirClient.create();
   });
 
   afterEach(async () => {
```

Comment:
We're replacing it to `function()` in some `afterEach` blocks and keeping it as `() =>` in others. It's a good idea to keep it consistent.

Reply:
I wonder whether it is related to the subtle differences regarding `this`?

Reply:
Yes, `this` is undefined in the fat arrow functions.
So, had to update beforeeach sections from fat arrow to `function()` instead.
However, we don't need to update aftereach since we don't use `this` in the aftereach section.

Still, to remain consistent, I've updated all the aftereach blocks now.